integration-tests/cypress/support/reactFlowCommands.js

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import '@testing-library/user-event';
+
+Cypress.Commands.add('dragAndDropNode', (sourceNodeTestId, targetNodeTestId) => {
+  cy.window().then((win) => {
+    const sourceNode = cy.getByTestId(sourceNodeTestId);
+    const targetNode = cy.getByTestId(targetNodeTestId);
+
+    targetNode.then((node) => {
+      const target = node[0];
+      const { x, y } = target.getBoundingClientRect();
+      sourceNode
+        .trigger('mousedown', {
+          button: 0,
+          force: true,
+          view: win,
+        })
+        .wait(400)
+        .trigger('mousemove', {
+          clientX: x,
+          clientY: y,
+          force: true,
+          view: win,
+        })
+        .trigger('mouseup', { view: win })
+        .wait(400);
+    });
+  });
+});
+
+Cypress.Commands.add('isNodeChildOf', (childNodeTestId, parentNodeTestId) => {
```

Comment:
It's more "isInside" that "is child of" (at least currently we can have sub-nodes which are only partially inside their parent).

Reply:
I renamed this command.
I don't know if this was the best way to do it, but I couldn't find any other way to easily know that a node is a subNode of another node.

Reply:
Ideally we should check that the tool had the expected *semantic* effect. In the context of a Cypress test the closest we can do is check that the dropped semantic element is now a child of the drop target *in the explorer*.